app/Http/Controllers/adminDashboard.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class adminDashboard extends Controller
+{
+    public function index(){
+        return view('admin.dashboard');
```

Comment:
Is this supposed to be "My Practicals" instead of Dashboard? Please refer to the Design

Reply:
yes, corrected